packages/calendar/react/src/components/confirmation-pane/confirmation-pane.tsx

```diff
@@ -37,7 +37,7 @@ const ConfirmationPane: React.FC<{
   }, [eventGroupId]);
 
   const { data: accounts } = trpc.account.me.useQuery();
-  const primaryAccount = accounts?.find((a) => a.isPrimary);
+  const primaryAccount = accounts?.find((a) => a.userIdPrimary);
```

Comment:
this will always return the first account, right? 

```suggestion
  const primaryAccount = accounts?.find((a) => a.id === a.userIdPrimary);
```
?

Reply:
Yes, this is flat out wrong. I'm not even sure why I didn't get a type error on this, there is no `userIdPrimary` on the account model. Only a `userPrimary` object, which I wanted to test like `Boolean(userPrimary)`. 🤔